Mathlib/SetTheory/Game/Basic.lean

```diff
@@ -899,6 +993,13 @@ theorem zero_lf_inv' : ∀ x : PGame, 0 ⧏ inv' x
     convert lf_mk _ _ InvTy.zero
     rfl
 
+/-- `inv' 0` has exactly the same moves as `1`. -/
+lemma inv'_zero : inv' 0 ≡ (1 : PGame) := by
```

Comment:
I know you were just following the existing lemmas but in my opinion, these lemmas about `inv' 0` should be removed. The purpose of `inv'` is only a temporary step to define inverses for positive numbers, and the full definition of inverse is in `instance : Inv PGame` below which is the actual definition of interest. There is no need for `inv' 0`. If you decide to keep this, I would recommend rewriting with `Identical.ext_iff` as in my previous suggestion for `inv'_one`.

Reply:
Admittedly `inv' 0 = 0` is more of an API curiosity, but we've got lemmas like `1 / 0 = 0` elsewhere in Lean. All the other lemmas on `inv'` seem at least potentially useful, since to prove things about `inv`, you have to case on the sign of your number, and both cases then unfold to it.

Reply:
`1 / 0 = 0` is fine, and the analog of that for the surreals is the theorem `inv_zero : (0 : PGame)⁻¹ = 0` found later in this file. But the proof of `inv_zero` doesn't use any theorem about `inv'` because `x⁻¹` is defined by 
```
noncomputable instance : Inv PGame :=
  ⟨by classical exact fun x => if x ≈ 0 then 0 else if 0 < x then inv' x else -inv' (-x)⟩
```
So the definition of `x⁻¹` (which is what we really want) only uses `inv'` in the non-zero cases; for 0 it bypasses `inv'` with `if x ≈ 0 then 0`. That's why I'm saying we don't need `inv' 0`.